src/site/scripts/components/profile/actionbutton.tsx

```diff
@@ -139,7 +157,8 @@ export class ActionButton extends React.Component<IActionButtonProps, IActionBut
         if (this.state.expanded) {
             this.setState({
                 confirmation: this.state.confirmation,
-                expanded: false
+                expanded: false,
+                delay: 0
```

Comment:
It looks like delay was used as the "countdown" for the confirmation button. I set delay to 0 to cancel the timer.